src/pages/Designer/Designer.js

```diff
@@ -0,0 +1,253 @@
+                height="xxsmall"
+                justify="center"
+                margin={{ left: 'xlarge' }}
+              >
+                <Text weight="bold" alignSelf="center" color="black">
+                  Open Designer
+                </Text>
+              </Box>
+            </Box>
+          </Box>
+          <IconCard
+            size="medium"
+            icon={<ClearOption size="xxlarge" color="DesignerYellow" />}
+            margin={{ left: 'xlarge' }}
+          />
+        </Stack>
+      </Box>
+      <Box
+        direction="column"
+        gap="medium"
+        margin={{ left: 'xlarge', bottom: 'xlarge' }}
+      >
+        <Box direction="row" margin={{ left: 'xlarge' }} gap="xlarge">
+          <Camera size="large" color="ImagerOrange" />
+          <Configure size="large" color="ThemerOrange" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }} direction="row">
+          <Box width="xxsmall"></Box>
+          <CloudUpload size="large" color="PublisherPink" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }} direction="row">
+          <Box width="xsmall"></Box>
+          <Selection size="large" color="SlidesBlue" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }}>
+          <BarChart size="large" color="TabularGreen" />
+        </Box>
+      </Box>
+
+      <Stack>
+        <Box
+          background="YellowBackground"
+          height="large"
+          margin={{
+            top: 'xlarge',
+          }}
+        ></Box>
+        <Box
+          margin={{
+            horizontal: 'xlarge',
+            bottom: 'xlarge',
+          }}
+        >
+          <Heading size="large" margin={{ top: 'xlarge' }}>
+            What is the Designer?
+          </Heading>
+          <Box width="large" margin={{ bottom: 'xlarge', top: 'none' }}>
+            <Paragraph fill={true} size="xxlarge">
+              A free, web-based WSIWG component editor, Grommet Designer is a
+              themeable experience designer baked on top of react and java
+              framework. See some examples from Grommet designers below.
+            </Paragraph>
+          </Box>
+          <Box
+            direction="row-responsive"
+            justify="center"
+            gap="large"
+            margin={{ vertical: 'xlarge' }}
+          >
+            <Box direction="column">
+              <Image src="Designer_1.svg" />
+              <Text alignSelf="center" color="altGrey">
+                Designer File Name
+              </Text>
+            </Box>
+            <Box direction="column">
+              <Image src="Designer_2.svg" />
+              <Text alignSelf="center" color="altGrey">
+                Designer File Name
+              </Text>
+            </Box>
+            <Box direction="column">
+              <Image src="Designer_3.svg" />
+              <Text alignSelf="center" color="altGrey">
+                Designer File Name
+              </Text>
+            </Box>
```

Comment:
Looks like a reusable functional component.

Reply:
It has been changed to a reusable component in the most recent push